packages/core/src/tools/errorFilter.ts

```diff
@@ -0,0 +1,41 @@
+import { Configuration } from '../domain/configuration'
+import { ErrorSource, RawError } from './error'
+import { clocksNow } from './timeUtils'
+import { ONE_MINUTE } from './utils'
+
+export type ErrorFilter = ReturnType<typeof createErrorFilter>
+
+export function createErrorFilter(configuration: Configuration, onLimitReached: (limitError: RawError) => void) {
+  let errorCount = 0
+  let isSendingLimitError = false
```

Comment:
This variable allows to send errors only while the `onLimitReached` callback is invoked, so it will already forbid subsequent errors from being sent if the customers filters the 'limit error'. I can rename it though, `allowNextError` seems fine if we actually reset it to `false` after using it.

Having a test based on the error message would be a bit more complicated because the message is not stored at the same place in Logs and RUM events.